nodes/mtdevice.py

```diff
@@ -1370,6 +1459,16 @@ def main():
                 print "xkf-scenario argument must be integer."
                 return 1
             actions.append('xkf-scenario')
+        elif o in ('-y', '--synchronization'):
+            sync_settings.append(get_synchronization_settings(a))
+            if sync_settings is None:
```

Comment:
Argh. Doesn't work since your testing a list and not the last element: it should be something like:
```python
new_sync_setting = get_synchronization_settings(a)
if new_sync_setting is None:
    return 1
sync_settings.append(new_sync_setting)
```